extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/serde/uriExtractors.scala

```diff
@@ -93,3 +95,11 @@ class PartitionLocsSeqURIExtractor extends URIExtractor {
     v1.asInstanceOf[Seq[(_, Option[String])]].flatMap(_._2).map(Uri)
   }
 }
+
+class IdentifierURIExtractor extends URIExtractor {
+  override def apply(v1: AnyRef): Seq[Uri] = v1 match {
+    case identifier: Identifier if isPathIdentifier(identifier.name()) =>
```

Comment:
It seems that we don't need to check the path again as the table code path has already done, which will bypass this code path

Reply:
I don't think so. If do like you say,  `create delta_ns.table1_delta` will throw `user [someone] does not have [create] privilege on [delta_ns/table1_delta], [write] privilege on [[table1_delta, table1_delta/]]`.

Reply:
OK